addons/l10n_es/data/template/account.tax-es_common.csv

```diff
@@ -546,7 +546,7 @@
 "account_tax_template_p_iva4_isp_bi","","VAT 4% reverse charge (investment goods)","","purchase","4% IG RC","IVA 4% ISP (bienes de inversión)","IVA 4% ISP (béns d'inversió)","4.0","percent","tax_group_iva_4","","sujeto_isp","","True","consu","","base","+mod303[30]||+mod303[12]","invoice","","100","",""
 "","","","","","","","","","","","","","","","","","tax","+mod303[31]","invoice","account_common_472","100","",""
 "","","","","","","","","","","","","","","","","","tax","-mod303[13]","invoice","account_common_477","-100","",""
-"","","","","","","","","","","","","","","","","","base","-mod303[40]||+mod303[14_purchase]","refund","","100","",""
+"","","","","","","","","","","","","","","","","","base","-mod303[40]||-mod303[14_purchase]","refund","","100","",""
```

Comment:
And L542, 537 not then?  (I did not check them all...)

Reply:
Not sure about this, I see that other taxes share the same tag but on the ticket only 4% tax was addressed. We need a functional check, maybe @chklop could help us here?

Reply:
@agr-odoo I see it is +mod303[14_purchase] everywhere, which makes it very tricky if you want to update it to -mod303[14_purchase]  (the idea should be that the base refunded amounts become negative amounts in the report normally, but I see that we have a formula that does sales - purchases (so the negative sign is in the formula and not the casilla)  @chklop